types/office-js/index.d.ts

```diff
@@ -35658,7 +35658,7 @@ declare namespace Excel {
          *
          * Creates a new comment with the given content on the given cell. An `InvalidArgument` error is thrown if the provided range is larger than one cell.
          *
-         * [Api set: ExcelApi 1.10]
+         * [Api set: ExcelApiOnline 1.1]
          *
          * @param cellAddress The cell to which the comment is added. This can be a Range object or a string. If it's a string, it must contain the full address, including the sheet name. An `InvalidArgument` error is thrown if the provided range is larger than one cell.
          * @param content The comment's content. This can be either a string or CommentRichContent object. Strings are used for plain text. CommentRichContent objects allow for other comment features, such as mentions. [Api set: ExcelApi 1.10 for string, ExcelApiOnline 1.1 for CommentRichContent object]
```

Comment:
Should this and other similar spots still call out both Api sets?

Reply:
Good call. Removed those references. I also gave the office-js-preview file the same treatment. This seems easier to read, so I want the front-and-center preview docs to have these changes.